packages/waku/src/vite-rsc/lib/render.ts

```diff
@@ -1,5 +1,5 @@
 import { renderToReadableStream } from '@vitejs/plugin-rsc/rsc';
-import { captureOwnerStack, type ReactNode } from 'react';
```

Comment:
nit, here is a wrong use case. https://react.dev/reference/react/captureOwnerStack#captureownerstack-is-not-available you shouldn't use named import for the API

Reply:
Ah, good to know. Indeed this works because Vite / Rollup / Rolldown is loose about cjs named exports, which I just happen to report to rolldown https://github.com/rolldown/rolldown/issues/5429. This would've been a build error on Webpack / Rspack.